recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,18 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,15 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if os.name == "posix":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir))
++elif os.name == "nt":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, "Library", "bin"))
```

Comment:
If pysdl2 is not respecting the PREFIX that it is installed to and searching for sdl2 there first by default, I consider this a bug in pysdl2.  If you have a patch that fixes this the default PREFIX search location, I think that is fair to put in here.  (However, that patch should really be put in as a PR upstream too.)

Barring such a patch, I think a no-patch version is also fine. As you say, it does work with conda. If the user is doing extraneous things with their environment, it is up to them to manage that via the environment variable. I don't think we can comprehensively predict all use cases.

Either of those two options would be fine with me.